lib/lighthouse/auth/client_credentials/service.rb

```diff
@@ -30,13 +30,14 @@ def initialize(token_url, api_scopes, client_id, aud_claim_url, rsa_key, service
         @tracker = AccessTokenTracker
         super()
       end
-      # rubocop:enable Metrics/ParameterLists
 
+      # rubocop:enable Metrics/ParameterLists
```

Comment:
Are the changes in this file intentional? I don't see a reason to keep these changes

Reply:
Yes, Rachal Cassity asked us to put those in as they were in the code before our changes and got removed in the process of all the changes done throughout this PR.  They exist currently in the master branch.